nova_api/__init__.py

```diff
@@ -244,7 +255,14 @@ def generate_api():
                         exc_info=True)
         sys.exit(3)
```

Comment:
Are we on the main thread? What do the standards (POSIX, ...) say about choosing exit() values? Check [https://www.man7.org/linux/man-pages/man3/exit.3.html](https://www.man7.org/linux/man-pages/man3/exit.3.html)

Reply:
Just expanded the search on this a little bit and found the FreeBSD standard exit codes which have been adopted by a number of C libraries. I'll adopt that!
https://www.freebsd.org/cgi/man.cgi?query=sysexits&apropos=0&sektion=0&manpath=FreeBSD%204.3-RELEASE&format=html